core/src/main/java/com/creditdatamw/zerocell/Reader.java

```diff
@@ -35,9 +38,23 @@ public ReaderBuilder(Class<T> clazz) {
             this.clazz = clazz;
         }
 
+        /**
+         * @deprecated use {@link #from(java.io.InputStream)} instead
+         */
+        @Deprecated
```

Comment:
Do not mark the File based API as deprecated. Besides keeping the API footprint flexible, Apache POI may process `java.io.File`s more efficiently